src/components/DotIndicatorMessage.js

```diff
@@ -71,14 +89,33 @@ function DotIndicatorMessage(props) {
                 />
             </View>
             <View style={styles.offlineFeedback.textContainer}>
-                {_.map(sortedMessages, (message, i) => (
-                    <Text
-                        key={i}
-                        style={[StyleUtils.getDotIndicatorTextStyles(isErrorMessage), ...props.textStyles]}
-                    >
-                        {message}
-                    </Text>
-                ))}
+                {_.map(sortedMessages, (message, i) =>
+                    isReceiptError(message) ? (
+                        <PressableWithoutFeedback
```

Comment:
@Gonals I'm working on `DotIndicatorMessage` TS migration and I while resolving conflicts I noticed ths logic change:

```ts
isReceiptError(message) ? (
                        <PressableWithoutFeedback
...
```

👇  With these lines `sortedMessages` is an array of strings, so I don't see how `isReceiptError` will ever return true in the code above, therefore the whole logic with `PressableWithoutFeedback` is unnecessary in my opinion 🤔

```ts
    const sortedMessages = _.chain(props.messages)
        .keys()
        .sortBy()
        .map((key) => props.messages[key])
        // Using uniq here since some fields are wrapped by the same OfflineWithFeedback component (e.g. WorkspaceReimburseView)
        // and can potentially pass the same error.
        .uniq()
        .map((message) => Localize.translateIfPhraseKey(message))
        .value();
```